third-party/cmake/BuildLuarocks.cmake

```diff
@@ -52,23 +52,41 @@ if(NOT MSVC)
   set(LUAROCKS_BUILDARGS CC=${HOSTDEPS_C_COMPILER} LD=${HOSTDEPS_C_COMPILER})
 endif()
 
+# Lua version, used with rocks directories.
+# Defaults to 5.1 for bundled LuaJIT/Lua.
+set(lua_version "5.1")
```

Comment:
why not uppercase?

```suggestion
set(LUA_VERSION "5.1")
```

Reply:
Changed to uppercase (also others).
IIRC this was an attempt to distinguish "internal"/local vars from others.